src/Tizen.NUI/src/internal/FrameProvider/FrameProvider.cs

```diff
@@ -37,7 +37,7 @@ internal class FrameProvider : IDisposable
         /// </summary>
         /// <param name="window">The window instance of Ecore_Wl2_Window pointer.</param>
         /// <exception cref="ArgumentException">Thrown when failed because of an invalid parameter.</exception>
-        /// <exception cref="Exceptions.OutOfMemoryException">Thrown when the memory is insufficient.</exception>
+        /// <exception cref="Applications.Exceptions.OutOfMemoryException">Thrown when the memory is insufficient.</exception>
```

Comment:
When there is not enough memory, `FrameProviderErrorFactory.GetException` will return `System.OutOfMemoryException`, not `Applications.Exceptions.OutOfMemoryException`.